### PR TITLE
Edit recipes

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

**Comment by sascala** [comment]
I've made some final edits based on your answers to my questions. (Thank you!)

**Comment by aquamatthias** [comment]
Thanks! LGTM. Please rebase.

**Comment by aquamatthias** [comment]
And also please squash your commits.

**Comment by sascala** [comment]
OK, @aquamatthias, I think I've done what you asked. I am a bit new to squashing -- let me know if I was not successful.

File: docs/docs/recipes.md [hunk 1]
```diff
@@ -158,8 +158,13 @@ See the detailed docs on
 ### Command (executor) health checks
 
 Command health checks are executed on the slave where the task is running.
+<<<<<<< ffddaf67e219b315ff56910eb3476bf6e6a8ef8f
```
Comment (aquamatthias) [inline docs/docs/recipes.md]: Please resolve this merge conflict.

**Comment by aquamatthias** [comment]
Hey @sascala you checked in a merge conflict. Do you have someone to ask how to handle a git rebase?

**Pull request merged**